package-lock.json

```diff
@@ -2749,7 +2749,7 @@
       "requires": {
         "@jobber/design": "^0.25.2",
         "@jobber/formatters": "^0.2.0",
-        "@jobber/hooks": "^1.11.0",
+        "@jobber/hooks": "^1.12.0",
```

Comment:
If you're just adding icons, it is unexpected to see dependency changes here - not sure if including this file in your PR was intentional?